packages/zoltan2/example/block/kokkosBlock.cpp

```diff
@@ -141,7 +142,7 @@ int main(int narg, char *arg[]) {
   // Check and print the solution.
   // Count number of IDs assigned to each part; compare to targetCount
 
-  Kokkos::View<globalId_t *> ids;
+  Kokkos::View<const globalId_t *, Kokkos::HostSpace> ids;
```

Comment:
Comment on line 156 below: You're assuming UVM there.  Consider the host mirror approach I discussed above.

Reply:
Comment on line 160 above: Please never do `&array(0)` or `&array[0]` or anything like that, ever, with any array data structure that has a `.data()` method.
 
1. Kokkos::View, like std::vector, has a `.data()` method; use it.
2. `&x[0]` with a zero-length std::vector may throw if you are using a checked STL.
3. `&x(0)` with a zero-length Kokkos::View will throw if Kokkos debug checking is on.

Reply:
Line 165 also assumes UVM.

Reply:
Switched to use data() method. Switched the views to HostSpace for now. See comments on prior. Will discuss with Karen to figure out how we want to structure this test. I'm not sure we want to work with any kernels or conversions here.